src/app/pages/register/register.component.html

```diff
@@ -0,0 +1,533 @@
+<div class="flex h-full w-full flex-col">
+  <!-- Header -->
+  <div class="flex items-center justify-between px-6 py-6 sm:px-12">
+    <a href="https://www.biocommons.org.au/">
+      <img
+        src="https://images.squarespace-cdn.com/content/5d3a4213cf4f5b00014ea1db/1689141619044-F67XDPQLP4PG6KY862VA/Australian-Biocommons-Logo-Horizontal-RGB.png?format=1500w&content-type=image%2Fpng"
+        class="auto h-16 min-w-44"
+        alt="Australian Biocommons Logo"
+      />
+    </a>
+    <div class="flex justify-between gap-6 font-light *:text-sky-900 sm:gap-12">
+      <a class="hover:text-sky-950" href="https://www.biocommons.org.au/about">
+        About
+      </a>
+      <a class="hover:text-sky-950" href="https://www.biocommons.org.au/">
+        FAQ
+      </a>
+      <a
+        class="hover:text-sky-950"
+        href="https://www.biocommons.org.au/contact-form"
+      >
+        Contact us
+      </a>
+    </div>
+  </div>
+
+  <!-- Select bundle step -->
+  @if (currentStep === 1) {
+    <div
+      class="flex w-full flex-1 flex-col items-center justify-start bg-gray-50 p-12"
+    >
+      <form
+        [formGroup]="bundleForm"
+        class="flex w-full max-w-4xl flex-1 flex-col gap-6"
+      >
+        <h1 class="text-4xl font-semibold text-gray-900">Access bundles</h1>
+        <p class="font-light text-gray-500">
+          Request access to a bundle you need for your research.
+        </p>
+
+        @if (
+          bundleForm.get("selectedBundle")?.invalid &&
+          bundleForm.get("selectedBundle")?.touched
+        ) {
+          <div class="text-sm text-red-500">
+            Please select a bundle to continue
+          </div>
+        }
+
+        <!-- Bundle selection cards -->
+        <div
```

Comment:
Can we generate these options from the bundles that are defined in the typescript component? Having them manually coded seems a bit clunky and will make updates to the bundles harder in future